conda-store-server/conda_store_server/_internal/utils.py

```diff
@@ -147,3 +149,33 @@ def callable_or_value(v, *args, **kwargs):
     if callable(v):
         return v(*args, **kwargs)
     return v
+
+
+def compile_arn_sql_like(
+    arn: str, allowed_regex: re.Pattern[AnyStr]
+) -> tuple[str, str]:
+    """Turn an arn into a string suitable for use in a SQL LIKE statement.
+
+    Parameters
+    ----------
+    arn : str
```

Comment:
Ah this is the response to my earlier question, let's add a comment in the schema then